app/pb_kits/playbook/pb_card/_card.jsx

```diff
@@ -36,13 +80,30 @@ const Card = ({
     [`highlight_${highlight.color}`]: highlight.color,
   })
 
+  // coerce to array
+  const leChildren = typeof(children) === 'object' && children.length ? children : [children]
+
+  const subComponentTags = (tagName) => {
+    return leChildren.filter((c) => {
+      return c.type && c.type.displayName === tagName
+    }).map((child, i) => {
+      return React.cloneElement(child, { key: `${tagName.toLowerCase()}-${i}` })
+    })
+  }
+
+  const nonHeaderChildren = leChildren.filter((child) => !child.type || child.type.displayName !== 'Header')
```

Comment:
This variable contains the non-Header children so that we may separate them from the header positionally. This means that even if the `Card.Header` tag is passed in as the last child, it will always be at the top..